pkg/app/app.go

```diff
@@ -332,14 +337,20 @@ func (a *App) Start() {
 		clusterURL = defaultClusterURLPreFix
 	}
 
-	daHandler := handler.NewHandler(a.broker, a.log.Logger, a.config.Broker, clusterURL, providers, rules)
+	daHandler := prometheus.InstrumentHandler(
+		"ansible-service-broker",
+		handler.NewHandler(a.broker, a.log.Logger, a.config.Broker, clusterURL, providers, rules),
+	)
 
 	if clusterURL == "/" {
 		genericserver.Handler.NonGoRestfulMux.HandlePrefix("/", daHandler)
 	} else {
 		genericserver.Handler.NonGoRestfulMux.HandlePrefix(fmt.Sprintf("%v/", clusterURL), daHandler)
 	}
 
+	defaultMetrics := routes.DefaultMetrics{}
+	defaultMetrics.Install(genericserver.Handler.NonGoRestfulMux)
```

Comment:
What is this doing?

Reply:
I guess we couldn't just add a metrics endpoint like we did with /v2 and the dev endpoints?

Reply:
https://github.com/kubernetes/apiserver/blob/master/pkg/server/routes/metrics.go#L31

We totally could, we get metrics that the apiserver sets up by default. Some have to do with apiserver audit events, those I would not want to set up on my own. But the others, (golang process metrics) we could set up on our own. Figured we already have apiserver in might as well use the stuff it provides for "free".